tests/unit/hosts/test_views.py

```diff
@@ -347,3 +347,39 @@ def test_create_challenge_host_and_team_with_no_data(self):
         del self.data['team_name']
         response = self.client.post(self.url, self.data)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+
+class DeleteChallengeHostFromTeamHimselfTest(BaseAPITestClass):
```

Comment:
Can you please rename this to `RemoveSelfFromChallengeHostTeamTest` to maintain consistency?